dist/Client.js

```diff
@@ -37,14 +38,22 @@ class APIError extends Error {
     this.body = body;
   }
 
+}
+
+exports.APIError = APIError;
+
+class ValidationError extends Error {
```

Comment:
I feel I wouldn't introduce validation of detector settings or annotation types. Those are things that are validated by our backend, so if the user put something wrong, I feel we should just surface the backend error.

The issue with validating those here is that if they change, you have to update the validation logic. So I'd keep this simple and rely on backend validation as much as possible.